tests/behaviour/expectations/behaviour.ts

```diff
@@ -25,4784 +25,4758 @@ export const expectations = flatten(
-//             Expect.Simple('uint256write', ['5', '6'], ['0', '0', '5', '6']),
-//           ]),
-//           File.Simple('structs', [
-//             Expect.Simple('createDefault', [], ['0', '0', '0']),
-//             Expect.Simple('createManual', ['1', '2', '3'], ['1', '2', '3']),
-//             Expect.Simple('writeMembers', ['1', '2', '3'], ['1', '2', '3']),
-//             Expect.Simple('references', ['1', '2', '3'], ['1', '2', '3']),
-//             Expect.Simple('input', ['3', '4', '5'], ['4', '5', '5']),
-//             Expect.Simple('output', ['3', '4', '5'], ['3', '4', '5']),
-//           ]),
-//         ]),
-//         new Dir('modifiers', [
-//           File.Simple('modifier', [
-//             Expect.Simple('f', ['90000', '0'], ['10000', '0']),
-//             Expect.Simple('f', ['110000', '0'], ['0', '0']),
-//           ]),
-//           File.Simple('modifierRecursive', [
-//             new Expect('modifier', [
-//               ['fact', ['5', '0'], ['120', '0'], '0'],
-//               ['called', [], ['6', '0'], '0'],
-//             ]),
-//           ]),
-//           File.Simple('modifierWithReturn', [
-//             Expect.Simple('returnFiveThroughModifiers', [], ['5']),
-//           ]),
-//           File.Simple('multipleModifiers', [
-//             new Expect('modifier', [
-//               ['openEvent', [], [], '0'],
-//               ['donate', ['238', '0'], ['238', '0'], '0'],
-//               ['donate', ['100', '0'], ['338', '0'], '0'],
-//               ['donate', ['50', '0'], null, '0', 'Value for donation must be bigger than 100'],
-//               ['balance', [], ['338', '0'], '0'],
-//               ['closeEvent', [], [], '0'],
-//               [
-//                 'donate',
-//                 ['500', '0'],
-//                 null,
-//                 '0',
-//                 'The event must be open in order to receive donations',
-//               ],
-//               ['balance', [], ['0', '0'], '0'],
-//             ]),
-//           ]),
-//         ]),
-//         new Dir('modifiers', [
-//           File.Simple('modifier', [
-//             Expect.Simple('f', ['90000', '0'], ['10000', '0']),
-//             Expect.Simple('f', ['110000', '0'], ['0', '0']),
-//           ]),
-//           File.Simple('multipleModifiers', [
-//             new Expect('modifier', [
-//               ['openEvent', [], [], '0'],
-//               ['donate', ['238', '0'], ['238', '0'], '0'],
-//               ['donate', ['100', '0'], ['338', '0'], '0'],
-//               ['donate', ['50', '0'], null, '0', 'Value for donation must be bigger than 100'],
-//               ['balance', [], ['338', '0'], '0'],
-//               ['closeEvent', [], [], '0'],
-//               [
-//                 'donate',
-//                 ['500', '0'],
-//                 null,
-//                 '0',
-//                 'The event must be open in order to receive donations',
-//               ],
-//             
```

Comment:
Did you try changing plain arrays to `toCairoUint256(x)` to test if constructor tests work?

Reply:
that function do same, diference is that it create the hex representation. Tests keep failing